packages/api-explorer/e2e/e2e.spec.ts

```diff
@@ -164,6 +164,9 @@ describe('API Explorer', () => {
         'Kotlin'
       )
       await expect(page).toMatchElement('h3', { text: 'Kotlin Declaration' })
+      await page.evaluate(() => {
```

Comment:
Observed this cause unpredictable behavior, new commit will move this to an `afterEach` within this test block.

Reply:
I can see the `afterEach` but this is still here. Is this intended?

Reply:
The most recent commit maintains the intended structure, there are two afterEach's that handle the clearing of local storage.